doc/tutorial/staramr-tutorial.ipynb

```diff
@@ -515,6 +691,26 @@
     "cut -f 1,3 out/summary.tsv | column -s$'\\t' -t"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Isolate ID       Plasmid Genes\n",
+      "GCF_001478105.1  IncI1, IncX1\n",
+      "GCF_001931595.1  None\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Show only Plasmid Genes\n",
+    "cut -f 1,4 out/summary.tsv | column -s$'\\t' -t"
```

Comment:
Let's just leave out showing the plasmid genes from the **Validation** section since we aren't comparing them to anything.

Reply:
yeah I agree, solved this in commit 2202e45